manifests/params.pp

```diff
@@ -9,21 +9,21 @@
     'Archlinux', 'Debian', 'RedHat': {
       $package_name     = 'haproxy'
       $global_options   = {
-        'log'     => "${::ipaddress} local0",
-        'chroot'  => '/var/lib/haproxy',
-        'pidfile' => '/var/run/haproxy.pid',
-        'maxconn' => '4000',
-        'user'    => 'haproxy',
-        'group'   => 'haproxy',
-        'daemon'  => '',
-        'stats'   => 'socket /var/lib/haproxy/stats'
+        'log'           => "${::ipaddress} local0",
+        'chroot'        => '/var/lib/haproxy',
+        'pidfile'       => '/var/run/haproxy.pid',
+        'maxconn'       => '4000',
+        'user'          => 'haproxy',
+        'group'         => 'haproxy',
+        'daemon'        => '',
+        'stats'         => 'socket /var/lib/haproxy/stats'
       }
       $defaults_options = {
-        'log'     => 'global',
-        'stats'   => 'enable',
-        'option'  => 'redispatch',
-        'retries' => '3',
-        'timeout' => [
+        'log'           => 'global',
+        'stats'         => 'enable',
+        'option'        => 'redispatch',
+        'retries'       => '3',
+        'timeout'       => [
```

Comment:
I don't think these spaces are necessary. The arrows here should be aligned with respect to the keys within the hash, not with the '=' around the hash. My previous comment was only related to lines 36 and 37.